ironic/drivers/modules/drac/raid.py

```diff
@@ -348,7 +385,7 @@ def _calculate_spans(raid_level, disks_count):
     """Calculates number of spans for a RAID level given a physical disk count
 
     :param raid_level: RAID level of the virtual disk.
-    :param disk_count: number of physical disks used for the virtual disk.
```

Comment:
Upstream reviewers will probably ask you to revert this change since it's outside the scope of this patch.

Reply:
Done